gsy_e_sdk/setups/Asset_API_scripts/rest_bid_offer_posting.py

```diff
@@ -0,0 +1,147 @@
+# flake8: noqa
+# pylint: disable=duplicate-code
+"""
+Template file for a trading strategy through the gsy-e-sdk api client using Rest.
+"""
+
+import os
+from time import sleep
+from gsy_e_sdk.aggregator import Aggregator
+from gsy_e_sdk.clients.rest_asset_client import RestAssetClient
+from gsy_e_sdk.utils import get_area_uuid_from_area_name_and_collaboration_id
+
+current_dir = os.path.dirname(__file__)
+ORACLE_NAME = "oracle"
+
+LOAD_NAMES = ["Load 1 L13", "Load 2 L21", "Load 3 L17"]
+PV_NAMES = ["PV 1 (4kW)", "PV 3 (5kW)"]
+STORAGE_NAMES = ["Tesla Powerwall 3"]
+AUTOMATIC = True
```

Comment:
How about naming it into the following or similar? This would make it more self explanatory. 
```suggestion
CONNECT_TO_ALL_ASSETS = True
```

Reply:
What do you think about my proposal? IMO AUTOMATIC could mean anything. If you want to keep the naming, please add an inline comment to explain what it means, thanks!